### DISCUSSION TITLE
Google Api Error: Invalid request - Prices are missing for the following billable countries: LB.

### DISCUSSION THREAD

**Discussion by pierre1590** [fastlane/fastlane#20780] [Q&A]
When submit an update to Play Store with expo with the command : ` eas submit -p android --latest`, I have the following result 
![UpdateSubmission](https://user-images.githubusercontent.com/3288682/197385950-496a66ae-9d52-440c-95de-c5fd6a52f82c.png)
How can I solve it?

**Comment by eonianvr** [comment]
found out that solution was to remove country Lebanon from Countries / Regions until resolved.
Console → Production → Country / Regions (tab) → then click on “Remove Countries / Regions”.
I’ll just check back once to re-enable Lebanon when this is resolved
